### PR TITLE
[SD-3755] Dont use redis keys

### PR DESCRIPTION
**Pull request by actionless**
https://dev.sourcefabric.org/browse/SD-3755

### CODE REVIEW

**Comment by actionless** [comment]
i am not sure what to do with the tests

**Comment by petrjasek** [comment]
imo it would good to have some helper methods for those sequences so you don't duplicate this `find_and_modify` query every time we use this, can you maybe introduce sequences module with a few methods to manipulate it?
then you could remove this `set_key` method and its tests

**Comment by actionless** [comment]
@petrjasek, like that?
```
def get_next_sequence_number(resource, id, min_number, max_number, field_name='sequence_number'):
```

**Comment by actionless** [comment]
@petrjasek done

**Comment by actionless** [comment]
so about the tests, should i do them for this new sequences helper?

**Comment by petrjasek** [comment]
right there should be a test for this new helper.
otherwise looks good.

File: superdesk/celery_app.py [hunk 1]
```diff
@@ -187,6 +187,7 @@ def _update_subtask_progress(task_id, current=None, total=None, done=None):
 
 def set_key(key, value=0, db=None):
     """
+    @TODO: deprecate this?
```
Comment (ioanpocol) [inline superdesk/celery_app.py]: Yes, please delete set_key method.

**Comment by ioanpocol** [comment]
:+1:  (except last 2 comments)

**Comment by actionless** [comment]
@ioanpocol @petrjasek done

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**